Week9 - homework SQL/LibrariesManagement.sql

```diff
@@ -0,0 +1,298 @@
+-- 3rd improvement: inserting into Books column AuthorEmail
+-- 4th improvement: dropping from Sales LibraryId and LibrarianId and adding BookLibraryId
+
+INSERT INTO Libraries
+VALUES('Luceafarul', 'Bulevardul Pacii', 'Iasi')
+INSERT INTO Libraries
+VALUES('Carusel', 'Bulevardul Eroilor', 'Bucuresti')
+INSERT INTO Libraries
+VALUES('Books and coffee', 'Anastasie Pana', 'Cluj')
+INSERT INTO Libraries
+VALUES('Lecturarium', 'Emil Racovita', 'Iasi')
+
+SELECT *
+FROM Libraries;
+
+ALTER TABLE Librarians
+ALTER COLUMN IsOnHoliday BIT;
+
+INSERT INTO Librarians
+VALUES('Andreea', '2020-05-10', 0, 1)
+INSERT INTO Librarians
+VALUES('Paul', '2016-07-30', 1, 2)
+INSERT INTO Librarians
+VALUES('Simona', '2021-01-24', 1, 4)
+INSERT INTO Librarians
+VALUES('Traian', '2023-01-15', 0, 4)
+INSERT INTO Librarians
+VALUES('Larisa', '2012-11-30', 0, 3)
+
+SELECT *
+FROM Librarians;
+
+INSERT INTO Books
+VALUES('Prietena mea geniala', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+INSERT INTO Books
+VALUES('Povestea noului nume', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+INSERT INTO Books
+VALUES('Scurta Istorie a Pamantului', 'Andrew', 'Knoll', 'nonfiction', 'a_knoll@gmail.com')
+INSERT INTO Books
+VALUES('Nascut in afara legii', 'Trevor', 'Noah', 'memoir', 'trevorrr@gmail.com')
+INSERT INTO Books
+VALUES('Din cer au cazut trei mere', 'Narine', 'Abgarian', 'fantasy', 'abgarian@gmail.com')
+INSERT INTO Books
+VALUES('Cei care pleaca si cei care raman', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+INSERT INTO Books
+VALUES('Ciuleandra', 'Liviu', 'Rebreanu', 'crime', 'rebreanu@postaromana.ro')
+INSERT INTO Books
+VALUES('Povestea fetitei pierdute', 'Elena', 'Ferrante', 'fantasy', 'ferrante@gmail.com')
+
+INSERT INTO Books_Libraries
+VALUES(1, 1)
+INSERT INTO Books_Libraries
+VALUES(1, 3)
+INSERT INTO Books_Libraries
+VALUES(2, 1)
+INSERT INTO Books_Libraries
+VALUES(2, 3)
+INSERT INTO Books_Libraries
+VALUES(3, 3)
+INSERT INTO Books_Libraries
+VALUES(4, 1)
+INSERT INTO Books_Libraries
+VALUES(5, 4)
+INSERT INTO Books_Libraries
+VALUES(6, 1)
+INSERT INTO Books_Libraries
+VALUES(6, 3)
+INSERT INTO Books_Libraries
+VALUES(7, 4)
+INSERT INTO Books_Libraries
+VALUES(8, 1)
+INSERT INTO Books_Libraries
+VALUES(8, 3)
+
+SELECT *
+FROM Books_Libraries;
+SELECT *
+FROM Libraries;
+SELECT *
+FROM Books;
+
+INSERT INTO Sales
+VALUES(114, 1)
+INSERT INTO Sales
+VALUES(98, 4)
+INSERT INTO Sales
+VALUES(321, 3)
+INSERT INTO Sales
+VALUES(987, 2)
+INSERT INTO Sales
+VALUES(89, 7)
+INSERT INTO Sales
+VALUES(56, 8)
+INSERT INTO Sales
+VALUES(347, 5)
+INSERT INTO Sales
+VALUES(563, 11)
+INSERT INTO Sales
+VALUES(97, 6)
+INSERT INTO Sales
+VALUES(224, 9)
+INSERT INTO Sales
+VALUES(90, 12)
+INSERT INTO Sales
+VALUES(542, 10)
+
+SELECT *
+FROM Sales;
+
+--exercise2
+--Display all the librarians together with the library name they work in ordered by librarian name and library name descending.
+SELECT L.Name, Lib.Name
```

Comment:
Both these selects should be merged into one that should do the ordering in the same time. Please have a look on my example once it's published.